Sources/ScheduledTasks.php

```diff
@@ -772,8 +773,18 @@ function ReduceMailQueue($number = false, $override_limit = false, $force_send =
 
 	// Send each email, yea!
 	$failed_emails = array();
+	$max_priority = 127;
+	$smtp_expire = ($modSettings['smtp_day_expiry'] ?? 3) * 60 * 60 * 24;
```

Comment:
If the value of the setting changes while there are unsent emails in the queue, it will mess up the calculations. I suggest ditching the setting and just hard coding the value. It's not the sort of thing that a forum admin would normally ever need to adjust anyway.

Reply:
I'm fine with that.